CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# How to contribute?
+
+First of all, thanks for taking the time to contribute! :tada::+1:
+
+Your contributions are important to make this project and event successful. 
+You can report mistakes and errors, propose a project, offer dataset and resources or create more content. 
+
+Whatever is your background, there is a way to contribute on this [GitHub repository](https://github.com/virtual-biohackathons/covid-19-bh20), 
+on our [mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/virtual-biohackathon)
+or other [related projects listed on our wiki page](https://github.com/virtual-biohackathons/covid-19-bh20/wiki).
+
+You can raise any issue relevant to this project/event publicly in the mailing list (google group), we will address and/or assess your change proposal as promptly as we
+can to make your participation easier.
+
+We have a [Code of Conduct](https://github.com/virtual-biohackathons/covid-19-bh20/blob/master/CODE_OF_CONDUCT.md) that applies to all the activities and projects involved in this event.
+For any organisation related queries or concern, you can directly reach out to the organisers Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) 
+and  Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp). 
+
+**ADD MISSING DETAILS**
+
+## How can I participate in this project/event?
+
+**ADD DETAILS**
+
+## How can I fix something on the repository?
+
+- Raise mistakes, error or missing information on this repository by opening a Pull Request
+  - Submit trivial fixes (for example, a typo, a broken link or an obvious error)
+  - Start work on a contribution that was already asked for, or that you’ve already discussed, in a mailing list
+  - A pull request doesn’t have to represent finished work. It’s usually better to open a pull request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. You can always add more commits later.
+  - [Read details for open a Pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request)
+
+## How can I join the mailing list and participate in an ongoing conversation?
+
+**ADD DETAILS**
+
+## How can join a project during the hackthon?
+- When you’ve found a project you’d like to contribute to, 
+do a quick scan to make sure that the project is suitable for accepting contributions. 
+**ADD DETAILS**
+
+## How can I offer dataset and other compute resources?
+
+**ADD DETAILS**
```

Comment:
People are pointing to data or offering compute resources via email. A dedicated wiki entry will include more details on the known/available data and compute resources.